src/link_header/index.jsx

```diff
@@ -4,25 +4,50 @@ import clsx from 'clsx';
 import { Link } from 'react-router-dom';
 
 
-const LinkHeader = (props) => {
-  const { children, to, reversed, ...cleanProps } = props;
-  const className = clsx('c-link_header', props.className, {
+const LinkHeader = ({
+  children,
+  to,
+  onClick,
+  reversed,
+  className: passedClassName,
+  ...cleanProps
+}) => {
+  const isClickable = to || onClick;
+
+  const className = clsx('c-link_header', passedClassName, {
     'is-reversed': reversed,
+    'is-clickable': isClickable,
   });
 
-  if (!to) return <span {...cleanProps} className={className}>{children}</span>;
+  let icon;
+  if (isClickable) {
+    const iconClass = clsx({
+      'icon-arrow_right': !reversed,
+      'icon-arrow_left': reversed,
+    });
 
-  const iconClass = clsx({
-    'icon-arrow_right': !reversed,
-    'icon-arrow_left': reversed,
-  });
-  const icon = <i className={iconClass} />;
+    icon = <i className={iconClass} />;
+  }
+
+  if (!to) {
```

Comment:
test what happens when both `to` and `onClick` are passed

Reply:
Good point. I changed it to pass on `onClick` to `Link`. Could make sense to use it in such way